pyrightconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/pyright/main/packages/vscode-pyright/schemas/pyrightconfig.schema.json",
     "typeshedPath": ".",
+    "venv": ".venv",
```

Comment:
Does this mean that it isn't going to work for people who prefer to name their venv something else? I usually name it `env`. It might not be a problem for me, because I don't run pyright locally very often.

Reply:
Yes, I think it will mean that. FWIW, I also prefer `env` as a name for my local virtual environments (it's fewer characters to type), but went with `.venv` here since it's the name recommended by our CONTRIBUTING docs:

https://github.com/python/typeshed/blob/ae0c9f9dadd72ee823d289416943a9109e24fe52/CONTRIBUTING.md?plain=1#L59

Reply:
`.venv` is also what's configured for flake8, gitignore, and isort.
mypy_test and regr_test use a temp directory anyway, but it's also using `.venv`

So it's pretty established that the standard for typeshed is to name the virtual environment folder `.venv`

Reply:
I would still prefer passing it in a command-line option instead of hard-coding it in the config.

Reply:
> I would still prefer passing it in a command-line option instead of hard-coding it in the config.

So would I, but if I'm reading the docs for pyright correctly, I don't think that's an option

Reply:
Apparently pyright has a `-v, --venv-path` option whose purpose is exactly this. From [the docs](https://github.com/microsoft/pyright/blob/main/docs/command-line.md):

> (3) This option is used in conjunction with configuration file, ... This allows a common config file to be checked in to the project and shared by everyone on the development team without making assumptions about the local paths to the venv directory on each developer’s computer.

Reply:
The `venv-path` option can be set via the command line (and I do in this PR, via pyright-action). But the `venv-path` option is meant to point to a directory in which there might be multiple different virtual environments, if I'm reading the pyright docs correctly, meaning it's useless to set the `venv-path` option on the command line unless you also set the `venv` option in the config file